doc/doc-support/python-interpreter-table/default.nix

```
@@ -0,0 +1,25 @@
+# From this folder, run with:
+#   `nix-build .`
+{ pkgs ? (import ../../.. { config = {}; overlays = []; }) }:
+let
+  inherit (pkgs.lib) getBin;
+  inherit (pkgs.lib.lists) map;
+  inherit (pkgs.lib.strings) concatStringsSep toJSON;
```

We don't really need to know the internals of lib

```suggestion
  inherit (pkgs.lib) getBin concatStringsSep toJSON;
```

Also map can be just used as a keyword